modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14692,6 +14695,7 @@ remove-all-items=Remove All Items
 remove-all-registered-fido2-authenticators=Remove All Registered FIDO2 Authenticators
 remove-all-translation-overrides=Remove all translation overrides
 remove-attachment=Remove Attachment
+remove-attribute-row=Remove attribute row
```

Comment:
Can we call it `Remove Attribute`. Where did "row" come from?

Reply:
I tought it was a good name... :sweat_smile: 
Removing it.

Reply:
No worries... You were right to add a label! 🥳 Otherwise it would be an accessibility issue. In the future we should all look closer at the designs and ask for labels to be added in the design phase. Then we can review all the language keys before we get to the review.